x11-misc/mugshot/mugshot-0.4.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eapi7-ver gnome2-utils
+
+MY_PV="$(ver_cut 1-2)"
```

Comment:
Used only once.

Reply:
So added directly to SRC_URI, if you mean that?

Reply:
Yes, that exactly.